### PR TITLE
Carpark model and endpoints

### LINKED ISSUE #18: Carpark Model [Booking Sys Func]
Setup carpark model and relevant endpoints in Django

### PR DESCRIPTION
**Pull request by Joel-Milligan**
Created a car park model following Lillian's ERD.
Also created endpoints for CRUD'ing car parks at `api/carparks` and `api/carparks/<id>`

Closes #18

### CODE REVIEW

**Comment by Joel-Milligan** [comment]
Has admin_settings and django-admin been removed by the authentication changes?

**Comment by zollf** [comment]
> 
> 
> Has admin_settings and django-admin been removed by the authentication changes?

yea i think i did it the other day, if u do need it add it back

File: app/core/settings.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
     'django_components',
     'rest_framework',
     'app.index',
+    'app.admin_settings',
```
Comment (zollf) [inline app/core/settings.py]: youll need to remove this

**Review by zollf — CHANGES_REQUESTED** [review]

File: app/core/settings.py (hunk 1 above)
Comment (zollf) [reply app/core/settings.py]: i moved this to app.admin now

File: app/core/settings.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
     'django_components',
     'rest_framework',
     'app.index',
+    'app.parking.apps.ParkingConfig'
```
Comment (zollf) [inline app/core/settings.py]: `app.parking`
You forgot the comma as well

**Review by zollf — CHANGES_REQUESTED** [review]

File: app/core/settings.py (hunk 2 above)
Comment (Joel-Milligan) [reply app/core/settings.py]: Thank you, I thought I was going crazy

**Comment by zollf** [comment]
`autopep8 --in-place --recursive .`

to fix lint

**Comment by Joel-Milligan** [comment]
Sorry about the failing tests and lints, I didn't know how to do it locally, but have now figured it out by looking at the actions, so shouldn't happen again

**Comment by Joel-Milligan** [comment]
> `autopep8 --in-place --recursive .`
> 
> to fix lint

Nice thanks

**Review by zollf — APPROVED** [review]
Good Work

**Pull request merged**